cms/static/cms/js/modules/cms.toolbar.js

```diff
@@ -149,6 +150,12 @@ var CMS = window.CMS || {};
                         e.stopPropagation();
                         var el = $(this);
 
+                        // close navigation once it's pressed again
+                        if (el.parent().hasClass(root) && open) {
+                            that.ui.body.trigger(that.click);
```

Comment:
apparently not, calling reset has some weird effects. There might be other events also intertwined